### DISCUSSION TITLE
PnP search webparts hosted on Teams shared channels SharePoint site

### DISCUSSION THREAD

**Discussion by SinghGurd** [microsoft-search/pnp-modern-search#2650] [General]
Can an external user accessing Teams shared channel SharePoint site view people search results? It does not appear to be working for us. We are trying to build a people search solution using PnP Search Web parts and host the search pages on a shared channel site. Search is working well when accessed by the resource tenant user. However, when an external user browses to search tab in shared channel, page loads Ok, but search is returning errors. I tried the get search REST API query directly in browser tab while in a shared channel browser session and query is returning 'An unknown error occurred.'.

**Comment by wobba** [comment]
This is an API limitation as you don't want to expose all internal users to any external user.
